src/main/java/Storage.java

```diff
@@ -0,0 +1,23 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private List<Task> myTasks = new ArrayList<>();
```

Comment:
Is it enough to just import ArrayList class? Since I think you can directly define the type of myTasks as ArrayList<Task>.